### DISCUSSION TITLE
Does copilot understand those Javascript framework?

### DISCUSSION THREAD

**Discussion by SAPNickYang** [community/community#17944] [Copilot Conversations]
Does any one know copilot also understand Java script framework?
Especially OpenUI5/SAPUI5.
Thanks.

**Comment by MrKrishnaAgarwal** [comment]
GitHub Copilot works with a broad set of frameworks and languages. The technical preview does especially well for Python, JavaScript, TypeScript, Ruby, Java, and Go, but it understands dozens of languages and can help you find your way around almost anything.

Have a great day, I hope it helps
